examples/node-agent-live/index.js

```
@@ -0,0 +1,110 @@
+        think: {
+          provider: {
+            type: "open_ai",
+          },
+          model: "gpt-4o-mini",
+        },
+      },
+    });
+
+    console.log("Deepgram agent configured!");
+
+    setInterval(() => {
```

ESLint

[error] 'setInterval' is not defined.

(no-undef)